src/utils/WellKnownUtils.ts

```diff
@@ -70,6 +75,16 @@ export function tileServerFromWellKnown(
     );
 }
 
+export function getEmbeddedPagesWellKnown(): IEmbeddedPagesWellKnown | undefined {
+    return embeddedPagesFromWellKnown(MatrixClientPeg.get().getClientWellKnown());
+}
+
+export function embeddedPagesFromWellKnown(
+    clientWellKnown?: IClientWellKnown | undefined,
```

Comment:
```suggestion
    clientWellKnown?: IClientWellKnown,
```

Reply:
I am not familiar enough with typescript to understand why this is safe to do. FYI: `tileServerFromWellKnown` also accepts `undefined`.

Reply:
The question mark makes it an optional parameter